nautobot_device_lifecycle_mgmt/views.py

```diff
@@ -239,6 +262,88 @@ class SoftwareLCMBulkImportView(generic.BulkImportView):
     default_return_url = "plugins:nautobot_device_lifecycle_mgmt:softwarelcm_list"
 
 
+class SoftwareSoftwareImagesLCMView(generic.ObjectView):
+    """Software Images tab for Software view."""
+
+    queryset = SoftwareLCM.objects.all()
+    template_name = "nautobot_device_lifecycle_mgmt/softwarelcm_software_images.html"
+
+    def get_extra_context(self, request, instance):
+        """Adds Software Images table."""
+        softwareimages = (
+            instance.software_images.annotate(device_type_count=count_related_m2m(SoftwareImageLCM, "device_types"))
+            .annotate(object_tag_count=count_related_m2m(SoftwareImageLCM, "object_tags"))
+            .restrict(request.user, "view")
+        )
+
+        if softwareimages.exists():
+            softwareimages_table = SoftwareImageLCMTable(data=softwareimages, user=request.user, orderable=False)
+        else:
+            softwareimages_table = None
+
+        paginate = {
+            "paginator_class": EnhancedPaginator,
```

Comment:
for the sake of curiosity, what are you using the `EnhancedPaginator` for?

Reply:
This is to keep UI and behavior consistency with Tables used in Views inheriting from `ObjectListView`.